osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -31,7 +34,7 @@
 
 namespace osu.Game.Overlays.Comments
 {
-    public class DrawableComment : CompositeDrawable
+    public class DrawableComment : CompositeDrawable, IHasPopover
```

Comment:
The comment shouldn't be the popover target, that makes the popover display in completely unexpected positions:

https://user-images.githubusercontent.com/22781491/196821619-742ddc33-54f2-459f-86ad-7d443b5529da.mp4

The "Report" text should be the target. That should be possible by moving the text to a separate (`ReportCommentLabel`) component that inherits `IHasPopover` and is added to the flow via `AddArbitraryDrawable`.